apps/dapp/package.json

```diff
@@ -16,8 +16,24 @@
     "build-with-stats": "tsc && BUILD_STATS=true vite build"
   },
   "dependencies": {
+    "@balancer-labs/sdk": "^0.1.43",
+    "@balancer-labs/sor": "^4.0.1-beta.16",
+    "@balancer-labs/typechain": "^1.0.0",
+    "@ethersproject/abi": "^5.7.0",
```

Comment:
had to add these deps. the Balancer SDK was having trouble when building without them